dispatch/modules/content/models.py

```diff
@@ -553,3 +553,42 @@ class Issue(Model):
     volume = PositiveIntegerField(null=True)
     issue = PositiveIntegerField(null=True)
     date = DateTimeField()
+
+class Poll(Model):
+    name = CharField(max_length=255)
+    question = CharField(max_length=255)
+    is_open = BooleanField(default=True)
+    show_results = BooleanField(default=True)
+
+    def save_answers(self, answers, is_new):
+        if is_new is False:
+            self.delete_old_answers(answers)
```

Comment:
Are db transactions in place in this request path? If not it makes sense to wrap this view in `@transaction.atomic` to avoid deleting old answers before knowing the new ones are successfully committed.